docs/search-omero.rst

```diff
@@ -66,6 +66,55 @@ Step-by-Step
 
 #. Click on the Browse link |image2|\ in one search result line of the last image (in the right-hand part of the centre pane) to navigate back to the main webclient.
 
+.. note::
+    Spaces, stars, question marks and ``^`` characters are to be avoided in the Keys.
+    If you already have these characters in Keys in your OMERO, 
+    then try to replace them with underscores if you want to 
+    use the Search functionality on them.
+    Spaces stars, question marks and ``^`` characters in Values 
+    are acceptable, but not preferable, see examples below.
+
+Key-Value pairs search examples
+-------------------------------
+
+Considering following setup of 12 separate images (image 1 through 12):
+
+1. Key ``GFP H2B`` with Value ``2 uM`` on image 1
+2. Key ``GFP^H2B`` with Value ``2 uM`` on image 2
+3. Key ``H2B`` with Value ``2`` on image 3
+4. Key ``H2B`` with Value ``4`` on image 4
+5. Key ``GFP-H2B`` with Value ``2-uM`` on image 5
+6. Key ``GFP-H2B`` with Value ``2 uM`` on image 6
+7. Key ``GFP_H2B`` with Value ``2_uM`` on image 7
+8. Key ``GFP^H2B`` with Value ``2^uM`` on image 8
+9. Image 9 named ``GFP`` with no Key-Value pairs
+10. Image 10 named ``uM`` with no Key-Value pairs
+11. Image 11 named ``H2B`` with no Key-Value pairs
+12. Image 12 named ``2`` with no Key-Value pairs
+
+Basic **Search** tab:
+   - ``GFP H2B:2 uM`` finds images 1,2,3,5,6,7,8,9,10. This is because the query is interpreted as ``GFP`` OR ``H2B:2`` OR ``uM``.
+   - ``"GFP H2B":2 uM`` throws an error. Do not use quotes around Key names!
+   - ``GFP H2B:"2 uM"`` finds images 1,2,5,6,7,8,9. This is because the query is interpreted as ``GFP`` OR ``H2B:2 uM`` which prevents finding of image 3 with Value ``2``.
+   - ``GFP^H2B:2 uM`` finds images 1,2,3,5,6,7,8,9,10. This is because the query is interpreted as ``GFP`` OR ``H2B:2`` OR ``uM``.
+   - ``H2B:2`` finds image 3.
+   - ``H2B:4`` finds image 4.
+   - ``GFP-H2B:2 uM`` finds images 1,2,5,6,7,8,10.
+   - ``GFP-H2B:2-uM`` finds images 5,6.
+   - ``GFP-H2B:"2-uM"`` finds images 5,6.
+   - ``GFP-H2B:"2 uM"`` finds images 5,6.
+   - ``GFP-H2B:"2_uM"`` finds images 5,6.
+   - ``GFP_H2B:2_uM`` finds image 7.
+   - ``GFP^H2B:2^uM`` finds images 1,2,3,5,6,7,8,9,10.
+   - ``GFP`` finds images 1,2,5,6,7,8,9.
+   - ``GFP`` with checkbox ``Name`` under ``Restricted by Field`` section checked finds image 9.
+
+**Advanced** tab:
+   - ``GFP^H2B:2^uM`` and ``GFP^H2B:2 uM`` throw an error in ``Advanced`` tab. This is because of the different interpretation of the ``^`` character between the basic ``Search`` and ``Advanced`` tabs.
+   - As there is no ``Name`` checkbox in the ``Advanced`` tab, use ``name:GFP`` instead, which finds image 9.
+
+The behaviour for the rest of the query examples in ``Advanced`` tab is the same as listed above for the basic ``Search`` tab.
+
```

Comment:
From this, it sounds like the `Advanced` tab is the same as the regular search, except that it has a couple of limitations. I can see from the walk-through above that you can do `AND..` etc searches with the Advanced search, but I wonder if it's worth describing the difference between them or showing some more examples of what the Advanced search offers? Not sure if it's possible to illustrate that with the Images listed above?

Reply:
> ...what the Advanced search offers?

@will-moore What larger topics would you like to see covered ? The logical operators are covered, in my mind, this is the main strength of the Advanced tab. Do you maybe mean fuzzy search ? What else ?